src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/NodeMonitoringFailoverPlugin.java

```diff
@@ -99,6 +100,7 @@ public NodeMonitoringFailoverPlugin(
     this.propertySet = propertySet;
```

Comment:
NIT: Line 78, i think all your arguments are small, you can put them all in one line